### ISSUE TITLE
Module `i3pystatus clock` could not be loaded

### ISSUE THREAD

**Issue by matjazmav** [ultrabug/py3status#1724]
Hey I can't make i3pystatus module to work. Can you please take a look?

py3status.log:
```
2019-02-24 11:18:06 INFO loading module "i3pystatus clock" from py3status.modules.i3pystatus
2019-02-24 11:18:06 INFO Module `i3pystatus clock` could not be loaded
2019-02-24 11:18:06 INFO No module named 'py3status.modules.i3pystatus'
```

py3status.conf:
```
order += "i3pystatus clock" 
i3pystatus clock {
    module = "clock"
    format = [('%a %b %-d %b %X', 'America/New_York'), ('%X', 'Etc/GMT+9')]
}
```

Version info:
```
$ pip3 list | grep i3pystatus
i3pystatus (3.35)
$ py3status --version
py3status version 3.7 (python 3.6.7)

$ python3 --version
Python 3.6.7

$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.2 LTS
Release:	18.04
Codename:	bionic
```

**Comment by matjazmav** [comment]
Also in modules folder, there is no file named i3pystatus:
```
/usr/lib/python3/dist-packages/py3status/modules$ ls -al | grep i3
-rw-r--r-- 1 root root 12889 jan  3  2018 i3block.py

```

**Comment by ultrabug** [comment]
Hi @matjazmav you are indeed using py3status version 3.7 but the `i3pystatus` module was released on 3.8.

Please note that latest version is 3.16 so I strongly encourage you to update your py3status mate :)

**Comment by lasers** [comment]
@matjazmav I see your `i3wm-conifg` branch. It looks like you updated your py3status, mate. :)

It looks like you just wanted to try out `i3pystatus` module using a docstring example. In case you liked `i3pystatus clock`, I suggest you to try out `clock` module too. Otherwise, I think we can close this now.  

If you have any questions or issues with other new modules, please let us know. Many thanks. :blush:
